src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/messaging/InboundEventsListener.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.messaging
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.awspring.cloud.sqs.annotation.SqsListener
+import mu.KotlinLogging
+import org.springframework.stereotype.Component
+
+private val log = KotlinLogging.logger {}
+private const val NOTIFICATION = "Notification"
+
+@Component
+class InboundEventsListener(
+  private val mapper: ObjectMapper,
+  private val inboundEventsService: InboundEventsService,
+) {
+
+  @SqsListener("education-and-work-plan", factory = "hmppsQueueContainerFactoryProxy")
+  internal fun onMessage(sqsMessage: SqsMessage) {
```

Comment:
Nice - thats at least 1 layer of json/objectmapper confusion removed 👍 
Thank you 👍